clang-tools-extra/docs/clang-tidy/checks/performance/lost-std-move.rst

```diff
@@ -0,0 +1,14 @@
+.. title:: clang-tidy - performance-lost-std-move
+
+performance-lost-std-move
+=========================
+
+The check warns if copy constructor is used instead of std::move().
```

Comment:
Please synchronize with statement in Release Notes. `This check` should be omitted.